File: projects/_archive/local-meeting-transcriber/Backend/src/api/Models/Meeting.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Backend.Models;
public class Meeting
{
    [Key] public int Id { get; set; }
    [Required] public string UserId { get; set; } = "";
    public string? Title { get; set; }
    public string AudioPath { get; set; } = "";
    public string? Transcript { get; set; }
    public string? DiarizedTranscript { get; set; }
    public string? Summary { get; set; }
    public DateTime CreatedUtc { get; set; } = DateTime.UtcNow;
}

File: projects/_archive/local-meeting-transcriber/Frontend/src/types/meeting.ts
export interface Meeting {
  Id: number;
  Title?: string;
  AudioUrl: string;
  Summary?: string;
  Transcript?: string;
  DiarizedTranscript?: string;
  CreatedUtc: string;
}

export interface NormalizedMeeting {
  id: string | number;
  title?: string;
  audioUrl: string;
  summary?: string;
  transcript?: string;
  diarizedTranscript?: string;
  createdUtc: string;
}
